x-pack/legacy/plugins/infra/public/components/inventory/toolbars/container_toolbar.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ToolbarProps } from './toolbar';
+import { WithSource } from '../../../containers/with_source';
+import { WithWaffleOptions } from '../../../containers/waffle/with_waffle_options';
+import { WaffleMetricControls } from '../../waffle/waffle_metric_controls';
+import { WaffleGroupByControls } from '../../waffle/waffle_group_by_controls';
+import { Toolbar } from '../../eui/toolbar';
+import { InfraSnapshotMetricType } from '../../../graphql/types';
+import { fieldToName } from '../../waffle/lib/field_to_display_name';
+
+export const ContainerToolbar = (props: ToolbarProps) => {
+  const options = useMemo(() => {
+    const CPUUsage = i18n.translate('xpack.infra.waffle.metricOptions.cpuUsageText', {
+      defaultMessage: 'CPU usage',
+    });
+
+    const MemoryUsage = i18n.translate('xpack.infra.waffle.metricOptions.memoryUsageText', {
+      defaultMessage: 'Memory usage',
+    });
```

Comment:
With a lot of these translations repeated between the different toolbar files with the same `id` do you think we could extract them out in to a `common` file or something to that effect, and share them?